install-instructions.php

```diff
@@ -249,8 +249,7 @@
 									<a href="https://owncloud.com/download/?ref=orgB" target="_blank" class="button orange primary mb30">Test Enterprise Edition free for 30 days</a>
 									<hr />
 									<div class="download-notes">
- 										<p><strong>NOTE</strong>: The appliances have been updated to the <a href="https://owncloud.org/blog/new-in-owncloud-9-1/">freshly released ownCloud 9.1</a>. </p>
-										<p><strong>NOTE</strong>: The appliances have been updated to the <a href="https://owncloud.org/blog/owncloud-server-9-0-released/">freshly released ownCloud 9.0</a> but have only gone through absolutely minimal testing. It is recommended to not use these for important data yet - giving them a trial run however and giving us feedback is greatly appreciated!</p>
+ 										<p><strong>NOTE</strong>: The appliances have been updated to <a href="https://owncloud.org/blog/introducing-owncloud-x/">ownCloud 10.0</a> but have only gone through absolutely minimal testing. It is recommended to not use these for important data yet - giving them a trial run however and giving us feedback is greatly appreciated!</p>
```

Comment:
I tried the VirtualBox appliance and it does work (I have added user, installed files_texteditor from Market, uploaded/created files...)
The things that tricked me going through the installation process were:
1) The installation on the console of the VM is actually a Univention thing. It asks you to set a password for "Administrator", which I did. Actually that is an account called "Administrator" for the Univention management stuff. When you finally get to ownCloud, the oC login is User owncloud password owncloud. That tripped me up for a while, because I thought that the installer would make an ownCloud admin account called "Administrator" wit the password that I had given.

2) The installer on the console tells you to go to https://10.49.215.40 (whatever the IP address of your VM bridged onto your local LAN happens to be. That results in an SSL error message in Firefox (below) and an outright refusal to connect by Chrome:
![owncloud10vboxuniventionsslerror](https://user-images.githubusercontent.com/1535615/27114813-fab6bc2c-50e5-11e7-9cef-beaed81a7dba.png)

Removing the https and just going straight to 10.49.215.40 (plain old http) works:
![owncloud10vboxuniventionwithouthttps](https://user-images.githubusercontent.com/1535615/27114836-2a5b4d9e-50e6-11e7-8ba2-3a90fb38f783.png)

I wonder where a "tricks for the Univention installer" guide should be put?